go/runtime/registry/host_rofl_mgmt.go

```diff
@@ -142,6 +188,76 @@ func (rh *roflHostHandler) handleBundleList(rq *rofl.BundleListRequest) (*rofl.B
 	}, nil
 }
 
+func (rh *roflHostHandler) handleVolumeAdd(rq *rofl.VolumeAddRequest) (*rofl.VolumeAddResponse, error) {
+	if err := rh.ensureComponentPermissions(runtimeConfig.PermissionVolumeAdd); err != nil {
+		return nil, err
+	}
+
+	// Determine labels, make sure to override origin as that is used for isolation.
+	labels := maps.Clone(rq.Labels)
+	maps.Copy(labels, rh.getBundleManagementLabels())
+
+	volume, err := rh.getVolumeManager().Add(labels)
```

Comment:
This reads really poorly, as I would assume we are adding labels. But instead, we are sometimes creating volume with the given labels. I was thinking if sequence of calls `First(labels) *Volume`,  `Create(labels) ID`, `Register(id)`, and `Remove(id)` on error is better, but maybe it would be better to create a unique name from labels. 
I would also store volume and bundle manager in the constructor, and call `rh.volumeManager.Add(...)`.

Reply:
The only reason why it reads poorly is that currently there are no other attributes to a volume except labels. As mentioned, later there will probably be other attributes, but I wanted to keep this simple for the initial version.

Reply:
I would still prefer name `Create` or `GetOrCreate`.

Reply:
Yeah `Create` makes sense.